drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -874,19 +1063,24 @@ int stm32_clock_control_init(const struct device *dev)
 	/* AHB/AXI/HCLK clock is SYSCLK / HPRE */
 	/* If freq increases, set flash latency before any clock setting */
 	if (new_hclk_freq > old_hclk_freq) {
-		LL_SetFlashLatency(new_hclk_freq);
+		LL_FLASH_SetLatency(LL_FLASH_LATENCY_6);
```

Comment:
Changes impacting H7 series should be made at least in a dedicated commit, ideally in a dedicated PR

Reply:
remove change related to non-stm32h7rs